src/AppBundle/Entity/Adherent.php

```diff
@@ -176,4 +176,54 @@ public function getUsername(): string
     public function eraseCredentials()
     {
     }
+
+    public function getPhone()
+    {
+        return $this->phone;
+    }
+
+    public function getGender()
+    {
+        return $this->gender;
+    }
+
+    public function getFirstName()
+    {
+        return $this->firstName;
+    }
+
+    public function getLastName()
+    {
+        return $this->lastName;
+    }
+
+    public function getBirthdate()
+    {
+        return $this->birthdate;
```

Comment:
I don't think it's needed to clone the instance [here](https://github.com/EnMarche/en-marche.fr/pull/77/files#diff-78fe6f3e331a599f3c301c3ee3626789R27) since this getter still returns a mutable object, what about using `\DatetimeImmutable` for this property?